frontend/src/metabase/admin/people/components/GroupDetail.jsx

```diff
@@ -41,128 +40,34 @@ const GroupDescription = ({ group }) =>
     </div>
   ) : null;
 
-@connect(null, {
-  deleteMembership,
-})
-export default class GroupDetail extends Component {
-  constructor(props, context) {
-    super(props, context);
-    this.state = {
-      addUserVisible: false,
-      text: "",
-      selectedUsers: [],
-      members: null,
-      alertMessage: null,
-    };
+const GroupDetail = props => {
+  const [addUserVisible, setAddUserVisible] = useState(false);
+  const [text, setText] = useState("");
+  const [selectedUsers, setSelectedUsers] = useState([]);
+  const [alertMessage, setAlertMessage] = useState(false);
+
+  const {
```

Comment:
nit: We almost everywhere destruct props in the function signature, e.g. `const GroupDetail = ({ currentUser, group })`